client/rpc/block_converter.go

```diff
@@ -0,0 +1,147 @@
+package rpc
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	cmn "github.com/tendermint/tendermint/libs/common"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+	"github.com/tendermint/tendermint/types"
+	"github.com/tendermint/tendermint/version"
+	"sync"
+	"time"
+)
+
+type CosmosResultBlock struct {
```

Comment:
Can we think of better type prefixes? Maybe even just remove `Cosmos*`.

Reply:
https://github.com/tendermint/tendermint/blob/master/rpc/core/types/responses.go#L28

Reply:
Done